setup.cfg

```diff
@@ -1,6 +1,6 @@
 [tool:pytest]
 # Look for tests only in tests directories.
-python_files = "insights/tests/*" "insights/parsers/tests/*" "insights/combiners/tests/*" "insights/archive/test.py"
+python_files = "insights/tests/*" "insights/parsers/tests/*" "insights/combiners/tests/*" "insights/web/tests/*" "insights/console/tests/*" "insights/archive/test.py"
```

Comment:
We don't have `web` or `console` packages in the core.  Is there a reason you added these back?